### PR TITLE
Doc: add a page about users of PROJ

### PR DESCRIPTION
**Pull request by rouault**
Equivalent of https://gdal.org/software_using_gdal.html

I've started this page as I needed thta information for an upcoming presentation for PROJ :-)

We should publicize it on the mailing list so that people can add themselves to the list

### CODE REVIEW

File: docs/source/users.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- `pyproj <https://pyproj4.github.io/pyproj>`_ Python interface to PROJ
+- `QGIS <http://www.qgis.org>`_ A cross platform desktop GIS.
+- `R <http://www.r-project.org>`_ A free software environment for statistical computing and graphics
+
+Proprietary licensed software
+-----------------------------
+
+- `FME <http://www.safe.com>`_  A GIS translator package. Includes a PROJ based transformer.
+
+Geodesic organizations
```
Comment (kbevers) [inline docs/source/users.rst]:
```suggestion
Geodetic organizations
```

File: docs/source/users.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+----------------------
+
+- `Agence for Data Supply and Infrastructure of Denmark (SDFI) <https://eng.sdfi.dk/>`_ 
+- `Land Information New Zealand (LINZ) <https://www.linz.govt.nz/>`_
+- `National Land Survey of Finland (NLS) <https://www.maanmittauslaitos.fi/en>`_
```
Comment (kbevers) [inline docs/source/users.rst:30]: You can for sure put Norway, Sweden and Iceland here too.

**Review by kbevers — COMMENTED** [review]
Good idea!

File: docs/source/users.rst (hunk 1 above)
Comment (rouault) [reply docs/source/users.rst]: fixed

File: docs/source/users.rst (hunk 2 above)
Comment (rouault) [reply docs/source/users.rst:30]: done

Comment (busstoptaktik) [reply docs/source/users.rst:30]: ... and s/Agence for.../Agency for.../

Comment (rouault) [reply docs/source/users.rst:30]: > ... and s/Agence for.../Agency for.../

fixed

**Pull request merged**